### PR TITLE
updates to dev docs content

### PR DESCRIPTION
**Pull request by RossBugginsNHS**
## Description

Support for collections and draw io diagrams.

### Draw IO

Direct support for draw IO diagrams, see /nhs-notify/user-guides/drawio.html

### Collections

Support for collections and page rendering. Example in git repositories - /nhs-notify/notify-repos.html

## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation

## Checklist

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [x ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -9,47 +9,53 @@ nav_order: 1
...
-- High level developer documentation for source code location, and build process.
-- Acts as the "root" for all separate public NHS Notify repositories.
-
-## What this repo does not contain
-
-- Any configuration of details of the Deployment phases or infrastructure of the NHS Notify Platform.
+> Please visit the [NHS Service Catalogue](https://digital.nhs.uk/services/nhs-notify) for:
+>
+> - NHS Notify web site
+> - onboarding guidance
```
Comment (m-houston) [inline docs/index.md:17]:
```suggestion
> - Onboarding guidance
```

File: docs/_includes/notify-repo-list.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<h2>Repository List</h2>
+
+<ul>
+{% for repo in site.notify-repos %}
+  <li>
+    <a href="https://www.github.com/NHSDigital/{{repo.repo-name}}">
```
Comment (m-houston) [inline docs/_includes/notify-repo-list.html:6]: The generated repo page is never linked to

Comment (RossBugginsNHS) [reply docs/_includes/notify-repo-list.html]: Updated to link to the repo pages

File: docs/_collections/_notify-repos/nhs-notify-web-platform.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+repo-name: nhs-notify-web-platform
+owners: ["rossbugginsnhs"]
+name: Notify Web Platform
+description: Notify Micro frontend web orchestration platform
+layout: nhs-notify-web-platform
```
Comment (m-houston) [inline docs/_collections/_notify-repos/nhs-notify-web-platform.md:6]:
```suggestion
layout: notify-repo
```

**Review by m-houston — APPROVED** [review]

File: docs/_collections/_notify-repos/nhs-notify-web-platform.md (hunk 3 above)
Comment (RossBugginsNHS) [reply docs/_collections/_notify-repos/nhs-notify-web-platform.md]: ta for that, good spot. Have committed that but it now wants re approval from you :-) cheers

**Pull request merged**